src/sagemaker_containers/_env.py

```diff
@@ -547,14 +546,17 @@ def __init__(self, resource_config=None, input_data_config=None, hyperparameters
             if k not in _params.SAGEMAKER_HYPERPARAMETERS
         }
 
-        sagemaker_region = sagemaker_hyperparameters.get(_params.REGION_NAME_PARAM, boto3.session.Session().region_name)
+        sagemaker_region = sagemaker_hyperparameters.get(_params.REGION_NAME_PARAM,
+                                                         boto3.session.Session().region_name)
 
         os.environ[_params.JOB_NAME_ENV] = sagemaker_hyperparameters.get(_params.JOB_NAME_PARAM, '')
         os.environ[_params.CURRENT_HOST_ENV] = current_host
         os.environ[_params.REGION_NAME_ENV] = sagemaker_region or ''
 
         self._hosts = hosts
-        self._network_interface_name = network_interface_name
+
+        self._network_interface_name = resource_config.get('network_interface_name', 'eth0')
```

Comment:
Why 'eth0' and not 'ethwe'? (In fact, why either at all? Could you add a comment?)